src/main.cc

```diff
@@ -47,7 +47,7 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       unsigned short v1, v2, v3, v4;
       if (!parse_version_string(argv[++i], &v1, &v2, &v3, &v4))
-        return print_error("Unable to parse version string");
+        return print_error("Unable to parse version string for FileVersion");
```

Comment:
Why is adding one more line a concern?

I'm confused. Why do you need a function taking a variable number of arguments? We only need one: the value `parse_version_string` attempted to parse.